fabric-mining-levels-v0/src/main/resources/fabric.mod.json

```
@@ -18,10 +18,8 @@
   "depends": {
     "fabricloader": ">=0.4.0",
     "fabric-api-base": "*",
-    "fabric-tag-extensions-v0": "*"
+    "fabric-tag-extensions-v0": "*",
+    "fabric-tool-attributes-v1": "*"
```

```suggestion
    "fabric-tool-attribute-api-v1": "*"
```
Since the `fabric.mod.json` in v1 resources uses the new name.

---

oh fuck right, I've gotta do this on object-builders too